scripts/release-helper.py

```diff
@@ -15,77 +15,77 @@
 PROJECT_ID = os.getenv("PIVOTAL_TRACKER_PROJECT_ID")
 
 if not API_TOKEN or not PROJECT_ID:
-    raise EnvironmentError(
```

Comment:
[pebble] All our Python scripts use four spaces for indentation instead of two, so we either need to update the spacing on all of them (in a separate commit in a separate PR) or we need to preserve this indentation in this script.

Reply:
I'll update